source/get_user_tweets.py

```diff
@@ -0,0 +1,16 @@
+import aiohttp
+import twint
+
+
+def download_user_tweets(username: str):
+    print(f"Downloading @{username} tweets\n")  # noqa
```

Comment:
I'd prefer using logging.   

```
from logging import info
info(f"Downloading @{username} tweets\n")
```

Also you will need to add logging configuration (if args.verbose   ...set_log_level(...INFO...))